### PR TITLE
Hue [core] Adding supervisor conf for Hue log listener command.

### PR DESCRIPTION
**Pull request by ranade1**
## What changes were proposed in this pull request?

- Add supervisor.conf file for the recently introduced hue log listener command

## How was this patch tested?

- manually, after docker creation

Please review [Hue Contributing Guide](https://github.com/cloudera/hue/blob/master/CONTRIBUTING.md) before opening a pull request.

### CODE REVIEW

**Review by wing2fly — APPROVED** [review]

File: tools/container/hue/supervisor-files/hue_loglistener_template [hunk 1]
```diff
@@ -0,0 +1,18 @@
+[program:hue_loglistener]
+directory=${HUE_HOME}
+command="${HUE_HOME}/desktop/core/src/desktop/loglistener.py"
```
Comment (amitsrivastava) [inline tools/container/hue/supervisor-files/hue_loglistener_template:3]: Does it also need the path to python3?

**Review by amitsrivastava — COMMENTED** [review]
How is this supervisor supposed to be used? Or is it just a template that can be used to create a service manually?

File: tools/container/hue/supervisor-files/hue_loglistener_template (hunk 1 above)
Comment (ranade1) [reply tools/container/hue/supervisor-files/hue_loglistener_template:3]: One the docker only python3.8 is installed.

Comment (amitsrivastava) [reply tools/container/hue/supervisor-files/hue_loglistener_template:3]: oh, I didn't realize that it is in the `container` directory. Sounds good.

**Review by amitsrivastava — APPROVED** [review]

**Pull request merged**